docs/source/reference/kubernetes/index.rst

```diff
@@ -3,27 +3,70 @@
 Running on Kubernetes
 =============================
 
-.. note::
-    Kubernetes support is under active development. `Please share your feedback <https://forms.gle/KmAtyNhEysiw2ZCR7>`_
-    or `directly reach out to the development team <http://slack.skypilot.co>`_
-    for feature requests and more.
-
 SkyPilot tasks can be run on your private on-prem or cloud Kubernetes clusters.
 The Kubernetes cluster gets added to the list of "clouds" in SkyPilot and SkyPilot
 tasks can be submitted to your Kubernetes cluster just like any other cloud provider.
 
-**Benefits of using SkyPilot to run jobs on your Kubernetes cluster:**
+Why use SkyPilot on Kubernetes?
+-------------------------------
 
-* Get SkyPilot features (setup management, job execution, queuing, logging, SSH access) on your Kubernetes resources
-* Replace complex Kubernetes manifests with simple SkyPilot tasks
-* Seamlessly "burst" jobs to the cloud if your Kubernetes cluster is congested
-* Retain observability and control over your cluster with your existing Kubernetes tools
+.. tab-set::
 
-**Supported Kubernetes deployments:**
+    .. tab-item:: For AI Developers
+        :sync: why-ai-devs-tab
 
-* Hosted Kubernetes services (EKS, GKE)
-* On-prem clusters (Kubeadm, Rancher)
-* Local development clusters (KinD, minikube)
+        .. grid:: 2
+            :gutter: 3
+
+            .. grid-item-card::  ✅ Ease of Use
+                :text-align: center
+
+                No complex kubernetes manifests - write a simple SkyPilot YAML and run ``sky launch``.
+
```

Comment:
I think that requires a little more work since we need to work on compressing our existing YAMLs (in terms of lines of code #3594) to get a good comparison against kubernetes manifests. E.g., [k8s vllm gemma manifest](https://cloud.google.com/kubernetes-engine/docs/tutorials/serve-gemma-gpu-vllm) is 65 lines, while SkyPilot gemma is 38 lines. We should do it as a part of the K8s blog post, marked it as a TODO for now.